### ISSUE TITLE
Container input support

### ISSUE THREAD

**Issue by buu700** [braintree/braintree-web-drop-in#159]
I was skimming some of the source code in lieu of documentation (see #158), when I noticed this: https://github.com/braintree/braintree-web-drop-in/blob/182bd39/src/dropin.js#L47

```
  if (!this._merchantConfiguration.selector) {
    analytics.sendEvent(this._client, 'configuration-error');
    callback(new DropinError('options.selector is required.'));
    return;
  }

  container = document.querySelector(this._merchantConfiguration.selector);
```

In Braintree v2, directly passing in `container` with a DOM element was supported (and my existing code depends on this). Would you be willing to add that option back in this module?

**Comment by crookedneighbor** [comment]
We decided not to support it in order to simplify the API.

Can you explain why your integration depends on it? Does the element not have an id property you can leverage? IE:

```js
braintree.dropin.create({
  authorization: auth,
  selector: '#' + myDomNode.id
}, callback);
```

**Comment by buu700** [comment]
We're using Braintree in an Angular component, so the most obvious way to access the intended DOM node is as a descendant of the root of the component.

Although, now that I think about it, while it's a bit of a hack, we can just generate a random GUID and set it as the ID to use for the selector.

**Comment by crookedneighbor** [comment]
This was released in version 1.1.0
